src/store/UI/index.js

```diff
@@ -21,6 +21,7 @@ export function toggleSettings () {
 
 
 function UIReducer (state = initialState, action) {
+  console.log('UI', action);
```

Comment:
fine. You are a plain ol' patty. I better not hear you complaining about es-lint telling you to get rid of `console.log`'s then :)